lineman/app/app.coffee

```diff
@@ -47,6 +47,7 @@ angular.module('loomioApp').controller 'AppController', ($scope, $filter, $rootS
     {path: '/inbox', component: 'inboxPage' },
     {path: '/groups', component: 'groupsPage' },
     {path: '/profile', component: 'profilePage'},
+    {path: '/email_settings', component: 'emailSettingsPage' },
```

Comment:
Which, after looking at it.. tells me the route should be /email_preferences as in..
https://www.loomio.org/email_preferences